app/src/main/java/io/appium/uiautomator2/model/api/gestures/ScrollModel.java

```diff
@@ -0,0 +1,30 @@
+package io.appium.uiautomator2.model.api.gestures;
+
+import androidx.test.uiautomator.Direction;
+
+import java.util.Arrays;
+
+import io.appium.uiautomator2.model.RequiredField;
+import io.appium.uiautomator2.model.api.BaseModel;
+import io.appium.uiautomator2.model.api.ElementModel;
+
+public class ScrollModel extends BaseModel {
+    public ElementModel origin;
+    public RectModel area;
+    @RequiredField
+    public String direction;
+    @RequiredField
+    public Float percent;
+    public Integer speed;
+
+    public ScrollModel() {}
```

Comment:
Codacy found an issue: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)